prow/cluster/pipeline-deployment.yaml

```diff
@@ -0,0 +1,40 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: prow-pipeline
+  namespace: default
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate 
+  selector:
+    matchLabels:
+      app: prow-pipeline
+  template:
+    metadata:
+      labels:
+        app: prow-pipeline
+    spec:
+      serviceAccount: prow-pipeline
+      containers:
+      - name: pipeline
+        image: gcr.io/k8s-prow/pipeline:v20190430-866dcd93f
+        args:
+        - --all-contexts
```

Comment:
What does this do?

Reply:
This means that all contexts defined in `--kubeconfig` will be provided to the controller, not only the `default`. Any of these build contexts can be then selected in the prow job spec.Cluster field, and the controller will create the resources in the given context.